### ISSUE TITLE
Extension intl uses old ICU version

### ISSUE THREAD

**Issue by korridor** [brefphp/bref#1534]
**Description:**

I use the PHP `intl` extension to format a number with currency. This does not work as expected, which is due to a bug in the ICU library, which was fixed a few years ago. So, I looked at which version Bref uses.

According to the constant `INTL_ICU_VERSION` ([docs](https://www.php.net/manual/en/intl.constants.php)) the version 50.2 is used.
ICU version `50.2` was released in Apr 11, 2019. (more than 4 years old) -> [Release note](https://github.com/unicode-org/icu/releases/tag/release-50-2)

**How to reproduce:**

Bref version: 1.7.23
Layers:
 - php-80
 - gd-php-80

PHP version: 8.0.28

**Comment by mnapoli** [comment]
Bref 1.7.23 is not the latest version, do you see the same problem with Bref v2?

**Comment by korridor** [comment]
I don't have a Bref v2 version in production, but I just tested it with the docker image `bref/php-81-fpm-dev:2` for the local setup and it uses the same version (`50.2`) there.

**Comment by korridor** [comment]
I just tested it with the PHP 8.1 image (`bref/php-81-fpm-dev:2`) as well and it also uses the same version.

**Comment by mnapoli** [comment]
Thank you, that's noted! We should upgrade the version in https://github.com/brefphp/aws-lambda-layers

PRs welcome!

**Comment by korridor** [comment]
Thanks! I just tried to update this package in the local `Dockerfile` with `yum`, but it says that it is already up-to-date. How do you normally update packages that are already bundled in the Amazon Linux 2 image?

**Comment by mnapoli** [comment]
Ah, this is probably Amazon Linux being outdated again :/ 

This is why we compile some dependencies, for example: https://github.com/brefphp/aws-lambda-layers/blob/main/php-80/Dockerfile#L72-L105 So that we get more up to date versions.

**Comment by fredericgboutin-yapla** [comment]
Yeah, it is so old that we cannot even use [NumberFormatter::CURRENCY_ACCOUNTING](https://www.php.net/manual/en/class.numberformatter.php#numberformatter.constants.currency-accounting) to format money amounts correctly. So on top of security issues, there is this.

BTW, we must use docker images directly so we cannot rely on AWS Lambda layers to get an updated version.

**Comment by mnapoli** [comment]
TBH PHP 8.0 has been EOL for almost a year now, it's no longer updated.

I don't think we'll upgrade the ICU version in the layers. I'll mark this as closed here to clean up the tracker, feel free to continue the discussion (and we can always reopen if needed).

**Comment by fredericgboutin-yapla** [comment]
@mnapoli it applies to ALL php versions...

**Comment by mnapoli** [comment]
Got it.
